code/game/machinery/shieldgen.dm

```diff
@@ -57,15 +58,15 @@
 
 
 	if (src.health <= 0)
-		visible_message("<span class='notice'>The [src] dissapates</span>")
+		visible_message("<span class='notice'>The [src] dissipates</span>")
```

Comment:
I'm serious. These changes aren't relevant to the PR, and we shouldn't have to roll back fixes as part of content changes.